pkg/watch/manager.go

```diff
@@ -130,8 +130,8 @@ func (wm *WatchManager) updateManager() (bool, error) {
 		return false, errp.Wrap(err, "could not filter pending resources, not restarting watch manager")
 	}
 
-	if len(readyToAdd) == 0 && len(removed) == 0 && len(changed) == 0 {
-		log.Info("no resources ready to watch and nothing to remove")
+	if wm.started == true && len(readyToAdd) == 0 && len(removed) == 0 && len(changed) == 0 {
+		log.Info("All changes are pending additions, not restarting watch manager")
```

Comment:
I think this log is fine, it's the previous check that needs to also record `started` 

""Watcher registry found changes, attempting to apply them" becomes
"Watcher registry found changes and/or needs restarting" and add `started`